front/lib/api/assistant/jit_actions.ts

```diff
@@ -370,3 +408,43 @@ export async function renderConversationForModelJIT({
     tokensUsed,
   });
 }
+
+async function listConversationFiles({
+  conversation,
+}: {
+  conversation: ConversationType;
+}) {
+  const fileAttachments: string[] = [];
+  for (const m of conversation.content.flat(1)) {
+    if (isContentFragmentType(m)) {
+      if (!m.fileId) {
+        continue;
+      }
+      fileAttachments.push(
+        `<file id="${m.fileId}" name="${m.title}" type="${m.contentType}" />`
+      );
+    } else if (isAgentMessageType(m)) {
+      for (const a of m.actions) {
+        if (isTablesQueryActionType(a)) {
+          const attachment = getTablesQueryResultsFileAttachment({
```

Comment:
The snippet is included here but we don't want it do we?

Reply:
indeed a bit weird my bad !